uobtheatre/productions/schema.py

```diff
@@ -337,7 +337,7 @@ def resolve_is_online(self, info):
         return False
 
     def resolve_duration_mins(self, info):
-        return self.duration().seconds // 60
+        return self.duration.total_seconds() // 60
```

Comment:
1. Can we make total_seconds a property?
2. Why don't we just make duration_mins a property on the performance then we don't need a custom resolver?